[docs][ydbops] add documentation for delay-between-restarts and nodes-inflight (#13301)
ydb-platform/ydb: ydb/docs/en/core/reference/ydbops/reference.md

- `--restart-duration <int>`: multiplied by `--restart-retry-number`, this gives the total duration in seconds for the maintenance operation. In other words, it is a promise to CMS that a single node restart will finish within given duration. Defaults to 60 (which makes the default CMS request duration 180 seconds in combination with the default value of `--restart-retry-number`)
- `--restart-retry-number <int>`: if restarting a specific node failed, repeat the restart operation this much times. Defaults to 3.
- `--cms-query-interval <int>`: how often to query for updates from CMS while waiting for new nodes. Defaults to 10 seconds.

> - `--nodes-inflight <int>`: the maximum number of nodes restarted at the same time.

Reviewer: ```suggestion
- `--nodes-inflight <int>`: the maximum number of nodes that are concurrently being restarted.
```

> - `--delay-between-restarts <duration>`: delay between two node restarts.

Reviewer: ```suggestion
- `--delay-between-restarts <duration>`: delay before initiating the next node restart.
```

Author: fixed
